docs/console/tutorials/authentication.md

```diff
@@ -0,0 +1,126 @@
+```
+
+:::note
+The Console **session lasts at most 24 hours** (and 10 hours in case of inactivity). After that, you will need to login again. When trying to use the Console after the session has expired, you will get the following error:
+
+```bash
+Console OAuth2 error: failed to request OAuth2 token: Server returned error response
+```
+
+:::
+
+  </TabItem>
+  <TabItem value="ash-api" label="Authenticate using the Ash Console API">
+
+To get an access token and a refresh token for the Ash Console API, you need to use the [OpenID Connect](https://openid.net/connect/) protocol:
+
+```bash
+curl -X POST -d "client_id=cf83e1357eefb8bd" \
+  -d "username=alice" \
+  -d "password=password4alice" \
+  -d "grant_type=password" \
+  "http://localhost:8090/realms/jeeo/protocol/openid-connect/token" | jq
+```
+
+The `access_token` has to be used in the `Authorization` header of Ash Console API requests:
+
+```bash
+curl "..." \
+  -H "Authorization: Bearer ${access_token}"
+```
+
+You can use the `refresh_token` to get a new `access_token` when the current one has expired:
+
+```bash
+curl -X POST -d "client_id=cf83e1357eefb8bd" \
+  -d "refresh_token=eyJhbGciOiJIUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI3NGY4MzgyYi00NTVkLTQ1Y2QtODA3NS05YmZkM2Q4ZjZlYWEifQ.eyJleHAiOjE2OTgyMDQ2NTIsImlhdCI6MTY5ODE2ODY1MiwianRpIjoiMmQ1MTA1YzgtNWRkNy00OWFjLThlZmItOTY4ZjlkNjBjYTY4IiwiaXNzIjoiaHR0cDovL2xvY2FsaG9zdDo4MDkwL3JlYWxtcy9qZWVvIiwiYXVkIjoiaHR0cDovL2xvY2FsaG9zdDo4MDkwL3JlYWxtcy9qZWVvIiwic3ViIjoiMjg3MDkwZWUtN2ZhZS00YmQ3LTkyNjMtZDY5MDk1OWVkYzI2IiwidHlwIjoiUmVmcmVzaCIsImF6cCI6ImNmODNlMTM1N2VlZmI4YmQiLCJzZXNzaW9uX3N0YXRlIjoiMWY1Yjk3MmMtMWZlMC00MjU0LTlmMTctNjNkZmM1YTZkYTViIiwic2NvcGUiOiJlbWFpbCBwcm9maWxlIiwic2lkIjoiMWY1Yjk3MmMtMWZlMC00MjU0LTlmMTctNjNkZmM1YTZkYTViIn0.lKbwKZwqMLqk-K6T_iLwT-b3zw80Q1cOSuAgy0wEt2w" \
+  -d "grant_type=refresh_token" \
+  "http://localhost:8090/realms/jeeo/protocol/openid-connect/token" | jq
+```
+
+:::note
+**An access token expire after 5 minutes** and a **refresh token after at most 24 hours** (and 10 hours in case of inactivity). After that, you will need to authenticate again. When trying to use the Console API after the access token has expired, you will get the following response:
+
+```json
+{
+  "detail": "Provided token is not valid",
+  "status": 401,
+  "title": "Unauthorized",
+  "type": "about:blank"
+}
+```
+
+:::
+
+:::tip
+An access token can also be obtain from the Ash CLI with the `auth show-token` command:
+
+```bash title="Command"
+ash console auth show-token
+```
+
+```bash title="Output"
+Showing access token for the Ash Console at http://localhost:8080
+Access token (valid):
+eyJ...WcA
+```
+
+If the access token is marked as `expired`, you can use the `auth refresh-token` command to get a new one:
+
+```bash title="Command"
+ash console auth refresh-token
+```
+
+```bash title="Output"
+Refreshing access token for the Ash Console at http://localhost:8080
+
+Access token refreshed successfully!
+```
+
+:::
```

Comment:
This should be in the `ash-cli` tab

Reply:
I don't think so because direct usage of access tokens is not part of the Ash CLI workflow

Reply:
It just feels weird to have `ash` commands in a "Using the API" tab when there is a "Using the CLI" tab IMO